TYPO3.Neos.NodeTypes/Resources/Private/TypoScript/Root.ts2

```diff
@@ -41,9 +41,11 @@ prototype(TYPO3.Neos.NodeTypes:Menu) {
 # Image TS Object
 prototype(TYPO3.Neos.NodeTypes:Image) {
 	maximumWidth = 2560
```

Comment:
I think this magic number should be set to NULL in this case the image w/h will be used automatically